inference-engine/samples/hello_classification/main.cpp

```diff
@@ -87,7 +87,7 @@ int main(int argc, char *argv[]) {
         // 2. Read a model in OpenVINO Intermediate Representation (.xml and .bin files) or ONNX (.onnx file) format
         CNNNetwork network = ie.ReadNetwork(input_model);
         if (network.getOutputsInfo().size() != 1) throw std::logic_error("Sample supports topologies with 1 output only");
-        network.setBatchSize(1);
+        if (network.getInputsInfo().size() != 1) throw std::logic_error("Sample supports topologies with 1 input only");
```

Comment:
As I know we pass only one image as an argument for this sample. What happens if batch in the model more than 1?

Reply:
I believe IE will throw an error, but this is the simplest sample and we would like to avoid redundant checks. Do you think it is necessary here?